protocol/signalfx/trace_zipkin.go

```diff
@@ -526,6 +530,13 @@ func (te *traceErrs) Append(err error) *traceErrs {
 	return out
 }
 
+// A parentSpanID of all hex 0s should be normalized to nil.
+func normalizeParentSpanID(parentSpanID **string) {
+	if *parentSpanID != nil && strings.Count(**parentSpanID, "0") == len(**parentSpanID) {
```

Comment:
I guess this is a fine way, handles varying lengths of 0s instead of just the case we're looking for. Seems complicated with the pointer to a pointer, why not just return the value it should be and always assign? Slightly more readable? i dunno. /shrug. I will say i looked at this and was like "hmm"..